backend/src/main/route/UserRouter.ts

```diff
@@ -0,0 +1,25 @@
+import { Router } from "express";
+import { UserController } from "../controller/UserController";
+
+export class UserRouter {
+    private userRouter = Router();
+    private userController = new UserController();
```

Comment:
Sorry, this was a mistake on my end, shouldn't instantiate with new since will be passed in the constructor.
Can just leave it as `private userController;`. I will fix on my PR